File: RepoTransBench/source_projects/Python/rust-lang_homu/public_tests/test_public_comments.py
import pytest

from homu import comments as comments_mod

def test_strip_mention_from_message():
    msg = "@botuser please test"
    assert comments_mod.strip_mention(msg) == "please test"
    msg = "  @dev hello Homu!**  "
    assert comments_mod.strip_mention(msg) == "hello Homu!**"

File: RepoTransBench/target_projects/Python/Rust/rust-lang_homu/public_tests/test_public_comments.rs
use homu_rust::comments;

#[test]
fn test_strip_mention_from_message() {
    let msg = "@botuser please test";
    assert_eq!(comments::strip_mention(msg), "please test");
    let msg = "  @dev hello Homu!**  ";
    assert_eq!(comments::strip_mention(msg), "hello Homu!**");
}